boards/shields/link_board_eth/doc/index.rst

```diff
@@ -0,0 +1,119 @@
++-----------------------+-----------------------+-----------------------------+
+| D9                    | INTn                  | 24 (Interrupt Output)       |
++-----------------------+-----------------------+-----------------------------+
+| D10                   | SPI CSn               | 34 (SPI Chip Select)        |
++-----------------------+-----------------------+-----------------------------+
+| D11                   | SPI MOSI              | 36 (SPI Serial Data Input)  |
++-----------------------+-----------------------+-----------------------------+
+| D12                   | SPI MISO              | 35 (SPI Serial Data Out)    |
++-----------------------+-----------------------+-----------------------------+
+| D13                   | SPI SCK               | 37 (SPI Serial Clock Input) |
++-----------------------+-----------------------+-----------------------------+
+| D14                   | not populated         |                             |
++-----------------------+-----------------------+-----------------------------+
+| D15                   | not populated         |                             |
++-----------------------+-----------------------+-----------------------------+
+
++-----------------------+-----------------------+-----------------------------+
+| Shield Connector Pin  | Function              | ENC424J600 Pin              |
++=======================+=======================+=============================+
+| A0                    | not connected         |                             |
++-----------------------+-----------------------+-----------------------------+
+| A1                    | not connected         |                             |
++-----------------------+-----------------------+-----------------------------+
+| A2                    | not connected         |                             |
++-----------------------+-----------------------+-----------------------------+
+| A3                    | not connected         |                             |
++-----------------------+-----------------------+-----------------------------+
+| A4                    | not connected         |                             |
++-----------------------+-----------------------+-----------------------------+
+| A5                    | not connected         |                             |
++-----------------------+-----------------------+-----------------------------+
+
+For more information about the link board ETH and ENC424J600:
+
+- `link board ETH Website`_
+- `ENC424J600 Website`_
+
+Programming
+***********
+
+Set ``-DSHIELD=link_board_eth`` when you invoke ``west build`` or ``cmake`` in your
+Zephyr application. For example:
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/net/zperf
+   :tool: all
+   :board: reel_board
+   :shield: link_board_eth
+   :goals: build
+
+.. _link board ETH Website:
+   https://www.phytec.de/link_board_eth/
```

Comment:
I get error 404 with this link..

Reply:
The page has not been created yet, should happen in the next few days.